Dockerfile

```diff
@@ -1,34 +1,57 @@
-#
+### ### ### ### ### ### ### ### ###
 # Builder layer
-#
+
 FROM node:6-alpine as ghost-builder
-RUN npm install --loglevel=error -g ghost-cli
 
-ENV GHOST_VERSION 1.17.1
-ENV GHOST_INSTALL /var/lib/ghost
-ENV GHOST_CONTENT /var/lib/ghost/content
-ENV GHOST_USER node
+RUN \
+    apk update && apk upgrade                           && \
```

Comment:
Useless here as you are not using apk in any other way and (as I written in the next comment) it is not a best practice, though, to upgrade the packages into a Docker.

Reply:
The idea here is to update existing package. 
When you say it's not best practice can you refer me some docs about this?

Reply:
Here an exemple: https://developers.redhat.com/blog/2016/02/24/10-things-to-avoid-in-docker-containers/